Client/Frontend/Browser/TabManager.swift

```diff
@@ -740,6 +740,10 @@ extension TabManager {
             let tab = addTab()
             selectTab(tab)
         }
+        // Always make sure there is a single normal tab.
+        if normalTabs.isEmpty {
+            addTab()
+        }
```

Comment:
The four lines above this (with the `count == 0`) are redundant now, so I'd remove them. You'll probably want to change your code to something like this:

```
if normalTabs.isEmpty {
    let tab = addTab()
    if selectedTab == nil {
        selectTab(tab)
    }
}
```